minor fixes (#49)
mbaudin47/hpcuqtraining: 2020/Courses/Metamodel/Introduction/steps-C-surrogate.tex

\item The goal of this course is to introduce the main surrogate methods used in 
steps C. 

\item Principles of surrogate models, cross-validation, coefficient of predictability, over-fitting.

> 

Author: we can probably talk about score instead of "coefficient of determination".

Let $g(\bx_v)$ and $\tilde{g}(\bx_v)$ the outputs of the 
model and metamodel on the validation sample.

> The $Q^2$ predictability coefficient is defined as the $R^2$ 

Author: ```suggestion
The $Q^2$ coefficient is defined similarly to the $R^2$
```
